### PR TITLE
Create ui-c8y-1021-25-0-add-gs-events-for-export-component-7503-graft-release-cd-7777.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-60216](https://cumulocity.atlassian.net/browse/MTM-60216)
Original PR: [7777](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7777)

[MTM-60216]: https://cumulocity.atlassian.net/browse/MTM-60216?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-25-0-add-gs-events-for-export-component-7503-graft-release-cd-7777.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Add GS events for export component  (#7503) [GRAFT][release/cd] (#7777)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-25-0-add-gs-events-for-export-component-7503-graft-release-cd-7777.md]: The Claude AI suggested to use the following title:
```suggestion
title: Added Gainsight events to export component
```

File: content/change-logs/application-enablement/ui-c8y-1021-25-0-add-gs-events-for-export-component-7503-graft-release-cd-7777.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60216
+version: 1021.25.0
+---
+Add GS events for export component  (#7503) [GRAFT][release/cd] (#7777)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-25-0-add-gs-events-for-export-component-7503-graft-release-cd-7777.md]: The Claude AI suggested to use the following body:
```suggestion
To improve the observability and monitoring capabilities of the export component, GS events have been added. The export component is responsible for exporting data from the system. With this change, relevant events occurring during the export process will now be logged and can be analyzed using the GS event monitoring infrastructure. This enhancement enables better tracking and debugging of the export functionality, allowing operators to gain insights into the behavior and performance of data exports.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by luja-c8y** [comment]
@rahulc8y @janhommes These are just a few gs events for the `export' component. I'm not sure this sort of thing should have release notes?

**Comment by BeateRixen** [comment]
@luja-c8y Please review and approve

File: content/change-logs/application-enablement/ui-c8y-1021-25-0-add-gs-events-for-export-component-7503-graft-release-cd-7777.md (hunk 2 above)
Comment (luja-c8y) [reply content/change-logs/application-enablement/ui-c8y-1021-25-0-add-gs-events-for-export-component-7503-graft-release-cd-7777.md]: I think:
```suggestion
Gainsight events were added to the export component to track and analyze data export operations.
```
will be enough.

**Review by luja-c8y — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-25-0-add-gs-events-for-export-component-7503-graft-release-cd-7777.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Added Gainsight events to export component
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-25-0-add-gs-events-for-export-component-7503-graft-release-cd-7777.md]:
```suggestion
title: Added Gainsight product experience tracking to export component
```

**Review by BeateRixen — APPROVED** [review]

**Comment by rahulc8y** [comment]
I don't think we should include information about Gainsight-related changes in our customer facing changelog. After reviewing the last 1 year's changelog, I did not find similar information from other c8y product

**Comment by luja-c8y** [comment]
> I don't think we should include information about Gainsight-related changes in our customer facing changelog. After reviewing the last 1 year's changelog, I did not find similar information from other c8y product

@rahulc8y @BeateRixen please approve revert PR: https://github.com/Cumulocity-IoT/c8y-docs/pull/2733

**Pull request merged**